### PR TITLE
Upgrade to PHP 7.3

### PR DESCRIPTION
**Pull request by ericgaspar**
-*Add PHP version 7.3*
-*Upgrade to upstream version*
-*Cleaning up*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Package_check results
---

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/ttrss_ynh%20PR82%20(ericgaspar)/)

### CODE REVIEW

**Review by rndmh3ro — APPROVED** [review]

File: manifest.json [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
   },
   "requirements": {
-    "yunohost": ">= 3.5"
+    "yunohost": ">= 3.8.1"
   },
   "multi_instance": true,
   "services": [
       "nginx",
-      "php5-fpm",
+      "php7.0-fpm",
```
Comment (SohKa) [inline manifest.json:22]: Hello ! Shouldn't the service be named "php7.3-fpm" ?

Comment (ericgaspar) [reply manifest.json:22]: I once tried to set it to php7.3-fpm and the installscript wasn't so happy about it. I think is it a declaration of services with no real use in terms of version specification.

**Comment by SimonMellerin** [comment]
Since ynh 4 this package is broken cause it does not install properly need php extensions (mysql, intl)
(may be it was installed by default with php5.6 before?)

This PR will fix it, thanks (see https://github.com/YunoHost-Apps/ttrss_ynh/issues/83)

File: scripts/upgrade [hunk 2]
```diff
@@ -20,6 +20,7 @@ domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
```
Comment (kay0u) [inline scripts/upgrade:23]: If you want this variable here (?) you should remove the line https://github.com/YunoHost-Apps/ttrss_ynh/blob/86c1f35ab8492dec124869365ff44bec50e4173e/scripts/upgrade#L143

File: scripts/install [hunk 3]
```diff
@@ -49,21 +49,15 @@ ynh_app_setting_set --app=$app --key=path --value=$path_url
...
-
-ynh_install_app_dependencies $pkg_dependencies
-
 #=================================================
 # CREATE A MYSQL DATABASE
 #=================================================
 ynh_script_progression --message="Creating a MySQL database..." --weight=2
 
 db_name=$(ynh_sanitize_dbid $app)
+db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
```
Comment (kay0u) [inline scripts/install:58]: This variable should be created by `ynh_mysql_setup_db`
```suggestion
```

File: scripts/install [hunk 4]
```diff
@@ -49,21 +49,15 @@ ynh_app_setting_set --app=$app --key=path --value=$path_url
...
 ynh_app_setting_set --app=$app --key=db_name --value=$db_name
-ynh_mysql_setup_db --db_user=$db_name --db_name=$db_name
+ynh_mysql_setup_db --db_user=$db_name --db_name=$db_name --db_pwd=$db_pwd
```
Comment (kay0u) [inline scripts/install:60]:
```suggestion
ynh_mysql_setup_db --db_user=$db_name --db_name=$db_name
```

File: conf/config.php [hunk 5]
```diff
@@ -8,7 +8,7 @@
 	define('DB_USER', "__DBNAME__");
 	define('DB_NAME', "__DBNAME__");
 	define('DB_PASS', "__DBPWD__");
-	define('DB_PORT', ''); // usually 5432 for PostgreSQL, 3306 for MySQL
+	define('DB_PORT', '3306'); // usually 5432 for PostgreSQL, 3306 for MySQL
```
Comment (kay0u) [inline conf/config.php:11]: Is it new? Why it was empty before? O.o

File: scripts/install [hunk 6]
```diff
@@ -49,19 +49,13 @@ ynh_app_setting_set --app=$app --key=path --value=$path_url
...
-
 #=================================================
 # CREATE A MYSQL DATABASE
 #=================================================
 ynh_script_progression --message="Creating a MySQL database..." --weight=2
 
 db_name=$(ynh_sanitize_dbid $app)
+db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
 ynh_app_setting_set --app=$app --key=db_name --value=$db_name
 ynh_mysql_setup_db --db_user=$db_name --db_name=$db_name
```
Comment (kay0u) [inline scripts/install]:
```suggestion
 db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
 
```

You can add this line here if you want

File: scripts/upgrade (hunk 2 above)
Comment (ericgaspar) [reply scripts/upgrade:23]: Do you mean that I don't need the db_pwd variable and I should remove it alltogether?

Comment (kay0u) [reply scripts/upgrade:23]: No, I mean we should only keep one of these lines

File: conf/config.php (hunk 5 above)
Comment (ericgaspar) [reply conf/config.php:11]: Empty at least since 2017 -.-

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM

**Review by kay0u — APPROVED** [review]
Thank you!

**Pull request merged**